iron-overlay-behavior.html

```diff
@@ -477,7 +478,6 @@
           this._focusNode.focus();
         }
       } else {
-        this._focusNode.blur();
```

Comment:
Hmm, is this bad? Like, will there be side effects because you used to be blurring this node and you're not anymore?